test/D2L.Security.OAuth2.Benchmarks/App.config

```diff
@@ -9,6 +9,14 @@
         <assemblyIdentity name="nunit.framework" publicKeyToken="96d09a1eb7f44a77" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-2.6.4.14350" newVersion="2.6.4.14350" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="log4net" publicKeyToken="669e0ddf0bb1aa2a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-1.2.15.0" newVersion="1.2.15.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="nunit.framework" publicKeyToken="2638cd05610744eb" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-3.6.1.0" newVersion="3.6.1.0" />

```

Comment:
yes. and it seems if I want to upgrade the NUnit(from 2.6.4 to 3.6.1) version for D2L.Security.OAuth2.TestFramework, then I also need to upgrade for the tests projects D2L.Security.OAuth2.Benchmarks, D2L.Security.OAuth2.IntegrationTests, D2L.Security.OAuth2.UnitTests, right?

And in order to upgrade NUnit version from 2.6.4 to 3.6.1, I also have to upgrade HttpMock from 1.1.5 to 1.3.3, log4net from 2.0.3 to 2.0.8.  as the dependencies NUnit->HttpMock->log4net.

I will upgrade all of them. 
The reason I want to do this is because I am upgrading Nunit from 2.6.4 to 3.6.1 for all test projects in Core. And one of the project depends on D2L.Security.OAuth2.TestFramework with old version of Nunit and HttpMock, which will cause the build failed.

I am not quite familiar with the usage of D2L.Security.OAuth2, if this will cause any other problems, please let me know. Thanks!